src/main/java/frc/robot/Constants.java

```diff
@@ -19,9 +19,12 @@ public class Constants {
     public static final double MK4I_STEER_INERTIA = 0.00001;
     public static final double MK4I_STEER_FRICTION_VOLTAGE = 0.25;
     public static final double MK4_WHEEL_RADIUS = Units.inchesToMeters(1.93);
+    public static final double WHEEL_COF = 1.2;
     public static final Slot0Configs MK4I_STEER_PID_CONSTANTS = new Slot0Configs()
             .withKP(50);
 
+    public static final double MAX_DRIVE_VELOCITY_MPS = 5.0;
```

Comment:
This is different for each robot. This should be calculated in the `CTREDrivetrainIO` based on the `ModuleConstants`.